src/server/routes/users.ts

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const controllers = require('../controllers.ts');
+const users = express.Router();
+
+users
+  .route('/create')
+  .post(controllers.createUser)
+
+users
+  .route('/:user_id')
+  .get(controllers.getOneUser)
```

Comment:
Did you happen to add the get user by email route? If not, I can add after merging.